### PR TITLE
slirp4netns: rename --create-sandbox to --enable-sandbox

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/rootless-containers/slirp4netns/issues/136

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -409,7 +409,7 @@ static void parse_args(int argc, char *const argv[], struct options *options)
 #define DISABLE_HOST_LOOPBACK -43
 #define NETNS_TYPE -44
 #define USERNS_PATH -45
-#define CREATE_SANDBOX -46
+#define ENABLE_SANDBOX -46
 #define _DEPRECATED_NO_HOST_LOOPBACK \
     -10043 // deprecated in favor of disable-host-loopback
```
Comment (AkihiroSuda) [inline main.c]: can we add `_DEPRECATED_CREATE_SANDBOX`?

Comment (giuseppe) [reply main.c]: do we already have any released version where `--create-sandbox` is present and it could be a breaking change?

Comment (AkihiroSuda) [reply main.c]: No, but Fedora 30 seems using packages built from git master?

Comment (giuseppe) [reply main.c]: good point.  I've amended the patch to include `--create-sandbox` as deprecated

**Pull request merged**